### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by mggcmatheus** [codewars/codewars.com#2845] [Q&A]
Hello, good afternoon, I understand that it may seem that there was plagiarism, but after solving a question, I saw the answers and submitted the answer of others, only with the intention of recording an improvement, but not with the intention of plagiarizing.

**Comment by Blind4Basics** [comment]
Hello, 
What about the alter account you created just to unlock solutions to the Kata you were interested in? _That's_ definitely "cheating",  don't you think?

**Reply by mggcmatheus** [reply]
First, I would like to thank you for the speed of response, very efficient.
About this problem of appearing to have created an account to collect solutions, I believe there is a mistake, because I am solving the Katas, together with several colleagues here from my work, and I believe that because there is more than one account of the same IP, it appears that was created by the same person.

Anyway, I really like the platform, and in no way did I intend to cheat the game, I would like my restriction to be reevaluated, and I promise to pay attention to submitting only codes of my own.

**Reply by Blind4Basics** [reply]
So mggcmatheus2 is NOT an alter account of yours? Posting the same solutions than your original account? You'll understand that there are some doubts, here... ;)
Restriction removed.
